LibSWBF2/Chunks/LVL/modl/modl.VBUF.h

```diff
@@ -16,9 +16,13 @@ namespace LibSWBF2::Chunks::LVL::modl
 		List<Vector3> m_Normals;
 		List<Vector3> m_Tangents;
 		List<Vector3> m_BiTangents;
+
 		List<Color4u8> m_Colors;
 		List<Vector2> m_TexCoords;
 
+		List<Vector3u8> m_BoneIndicies;
+		List<Vector3> m_Weights;
+
 	public:
```

Comment:
I'm not sure how to work in the ```SKIN``` chunk, after ignoring it and indexing directly into ```BMAP``` I got steadily correct results with the importer.  The unmunger doesn't read it at all, but there are is one skinned model, the sarlacc tentacle from the conversion pack's Dune Sea lvl, that does not load properly with this library and the unmunger.  That indicates it likely uses some older format that does require ```SKIN```s to be read.

Reply:
Possible. Question is if it worth the effort.